api/test/pact/constants/behaviors.ts

```diff
@@ -0,0 +1,6 @@
+export const SUCCESS = {};
```

Comment:
I think this should be `behaviors.spec.ts` so it's not compiled into the code. It's purely for testing.

Reply:
Actually, "behaviours" seeing as we're not American. :)

Reply:
@ficklephil are you happy for me to change the filename?

Reply:
@connorpgpmcelroy Yep go for it :)